packages/plugins/rubygem-fugit/rubygem-fugit.spec

```diff
@@ -0,0 +1,69 @@
+# template: default
+%global gem_name fugit
+
+Name: rubygem-%{gem_name}
+Version: 1.8.1
+Release: 1%{?dist}
+Summary: time tools for flor
+License: MIT
+URL: https://github.com/floraison/fugit
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+# start specfile generated dependencies
+Requires: ruby
+BuildRequires: ruby
+BuildRequires: rubygems-devel
+BuildRequires: rubygem(et-orbi) >= 1.2.7
+BuildRequires: rubygem(et-orbi) < 2.0.0
+BuildRequires: rubygem(raabro) >= 1.4.0
+BuildRequires: rubygem(raabro) < 2.0.0
```

Comment:
I wonder why this is being generated. I thought we only did that for gems with native extensions, which fugit doesn't appear to have.

It now means the other packages need to exist before this gem can be packaged.

Reply:
Ah ok. So what we now have with EL8 is that runtime requirements are automatically added. Build time dependencies are there if we need to actually run the package in steps like `%build` or `%check`, but here we don't so you can leave it out.